src/test/java/io/getunleash/DefaultUnleashTest.java

```diff
@@ -50,7 +50,7 @@ public void should_evaluate_all_toggle_with_context() {
         when(contextProvider.getContext()).thenReturn(UnleashContext.builder().build());
 
         List<EvaluatedToggle> toggles = sut.more().evaluateAllToggles();
-        assertThat(toggles).hasSize(2);
+        Assertions.assertThat(toggles).hasSize(2);
```

Comment:
Bring the static import back to improve readability

Reply:
Yup, will do